google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -2573,6 +2573,8 @@ def insert_async table_id, skip_invalid: nil, ignore_unknown: nil, max_bytes: 10
 
         protected
 
+        # rubocop:disable Lint/SuppressedException
```

Comment:
Instead of disabling this lint, put a comment inside the rescue clause, like:

```ruby
rescue Google::Cloud::AlreadyExistsError
  # Do nothing if it already exists
end
```

Reply:
Done, thanks!